Classes/Command/FetchCommand.php

```diff
@@ -2,6 +2,10 @@
 
 namespace Chili\Command;
 
+use Chili\Utility\Downloader;
+use Chili\Utility\General;
+use Chili\Utility\WebService;
```

Comment:
To stay in line with the rest of the code, no 'use statements', use fully qualified names..